tools/cacheload/cacheload.go

```diff
@@ -270,10 +270,11 @@ func main() {
 			writeQPSCounter.Inc()
 
 			if *readQPS > 0 {
+			outer:
 				for i := 0; i < readsPerWrite; i++ {
 					select {
 					case writtenDigests <- d:
-						break
+						break outer
```

Comment:
Ditto here, do you mind breaking out of the outer loop using a control variable instead of a labelled-for-loop?